homeassistant/util/ssl.py

```diff
@@ -112,21 +114,6 @@ def client_context(
     return _client_context(ssl_cipher_list=ssl_cipher_list)
 
 
-# Create this only once and reuse it
-_DEFAULT_SSL_CONTEXT = client_context()
-_DEFAULT_NO_VERIFY_SSL_CONTEXT = create_no_verify_ssl_context()
-
-
-def get_default_context() -> ssl.SSLContext:
-    """Return the default SSL context."""
-    return _DEFAULT_SSL_CONTEXT
-
-
-def get_default_no_verify_context() -> ssl.SSLContext:
-    """Return the default SSL context that does not verify the server certificate."""
-    return _DEFAULT_NO_VERIFY_SSL_CONTEXT
```

Comment:
We can't delete these methods as they are the public api

Reply:
They are not deleted, just moved a bit upward